### PR TITLE
Bengio Wave 11: 6L/256d depth extension with FourierPE + T_max=50

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

Adding a 6th transformer layer (6L vs current 5L baseline) should increase model capacity and improve prediction of wall-shear components, which remain the dominant binding constraint (wsy=8.73%, wsz=10.58% vs targets 3.65%, 3.63%). The 5L→6L depth increase previously showed improvement: each layer adds cross-attention capacity for learning fluid boundary-layer gradients. We keep hidden_dim=256 and heads=4 constant to isolate the depth effect. T_max=50 cosine schedule is applied as validated.

Previous depth scaling evidence: 4L→5L (PR #174) gave −0.254pp on abupt (7.2091% → 6.9549%). We test whether 5L→6L continues this trend.

## Instructions

Run a single 50-epoch experiment with 6 transformer layers, keeping all other hyperparameters identical to the current baseline:

```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 6 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --grad-clip-norm 1.0 \
  --no-compile-model \
  --wandb-group bengio-wave11 \
  --wandb-name alphonse-6l256d-tmax50-wave11 \
  --kill-thresholds "89084:val_primary/abupt_axis_mean_rel_l2_pct<13.0,178169:val_primary/abupt_axis_mean_rel_l2_pct<11.0,445350:val_primary/abupt_axis_mean_rel_l2_pct<6.9549"
```

**Kill gates:**
- Step 89,084 (ep5): must be below 13.0% — kill if NOT below
- Step 178,169 (ep10): must be below 11.0% — kill if NOT below
- Step 445,350 (ep25): must be below 6.9549% (current baseline) — kill if NOT below

**Report:** Post W&B run ID and best `val_primary/abupt_axis_mean_rel_l2_pct` at ep5, ep10, ep25, and final best. Also report `val_primary/wall_shear_y_rel_l2_pct` and `val_primary/wall_shear_z_rel_l2_pct` at each gate.

## Baseline

Current best (PR #174, alphonse, run `vu4jsiic`, ep~45.3, step 807,025):

| Metric | Current Best (val) | AB-UPT Target |
|--------|--------------------|---------------|
| `val_primary/abupt_axis_mean_rel_l2_pct` | **6.9549** | 4.51 |
| `val_primary/surface_pressure_rel_l2_pct` | 4.5644 | 3.82 |
| `val_primary/volume_pressure_rel_l2_pct` | 3.9361 ✓ | 6.08 |
| `val_primary/wall_shear_y_rel_l2_pct` | 8.7345 | 3.65 |
| `val_primary/wall_shear_z_rel_l2_pct` | 10.5766 | 3.63 |

Architecture: 5L/256d/4H + FourierPE + no-EMA + T_max=50 + lr=3e-4 + gc=1.0

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #426 because Closing for non-compliance: 30-minute ACK deadline expired with zero response. PR was created 2026-05-02T16:54Z; no ACK comment posted by ~17:24Z deadline. This is alphonse's second consecutive non-ACK failure (Wave 10 PR #413 also closed for zero ACK). Hypothesis (6L/256d depth extension + T_max=50) remains valid and will be reassigned to a compliant student in Wave 12..
